app/routes/teams.rb

```diff
@@ -17,6 +17,10 @@ class Teams < Core
         locals = {
           tag: params["q"],
           teams: teams.paginate(page: page, per_page: 10),
+          member_of: current_user.teams.ids + current_user.managed_teams.ids,
```

Comment:
Interesting. I didn't notice this before, but member_of includes more than just teams you're a member of. Is there a better name for this? If not, that's fine. Just thinking out loud.